docs/user-guide/update-demo/index.md

```diff
@@ -13,11 +13,11 @@ here](https://github.com/kubernetes/kubernetes.github.io/tree/{{page.docsbranch}
 
 ### Step Zero: Prerequisites
 
-This example assumes that you have forked the repository and [turned up a Kubernetes cluster](/docs/getting-started-guides/):
+This example assumes that you have forked the docs repository and [turned up a Kubernetes cluster](/docs/getting-started-guides/):
 
 ```shell
-$ cd kubernetes
-$ ./cluster/kube-up.sh
+$ git clone https://github.com/kubernetes/kubernetes.github.io
+$ cd kubernetes.github.io

```

Comment:
Agree on linking to specific branch. Will fix that in next patch.

afaik there is no easy way to just copy the update-demo folder unless we upload it as an independent archive somewhere. Currently the demo requires you to run kubectl proxy on a whole directory so it's not as easy as just linking to 2 files.

For the kubectl create commands we can use kubectl create -f http://location-to-yaml-file.yaml but let's do that as separate patch. 

Note though: This patch is only about fixing broken stuff not about improving the update-demo docs in general. I don't want it to get bloated. Currently update docs are broken because doc location moved from kubernetes repo to kubernetes.github.io repo and this should get fixed first.